build.sh

```diff
@@ -299,11 +299,11 @@ if [[ "$WATTSI_RESULT" -ne "0" ]]; then
     echo "The correct line numbers for errors in the $HTML_SOURCE/source file are shown below:"
     echo
     cat $HTML_TEMP/wattsi-output.txt | grep -v '^$' # trim blank lines
```

Comment:
> Oh, sorry, I see that we output the wrong line numbers on line 295. But we should probably output both? So keep the "Found fatal errors. Stopping" inside the if [[ "$WATTSI_RESULT" -ne "0" ]], and move lines 299-301 outside of it?

I vaguely recall that back in September when I was first hacking on this, I found that if we always output the error report for the build against the raw source, it always emits errors, because there’s at least one case where there’s a funky `dfn` value that I guess somehow gets corrected or normalized by wattsi such that it doesn’t emit the error when run against the pre-processed source.

But I think rightly we should fix that in the source that we can actually run wattsi against the raw source without it reporting errors. Otherwise, those error messages are anyway always going to be tossed into the output if there are other actual fatal errors that cause wattsi to exit non-zero.